LINUX/final-patches/vanilla--vmxnet3--10000--99999

```diff
@@ -0,0 +1,97 @@
+diff -ruN a/vmxnet3/vmxnet3_drv.c b/vmxnet3/vmxnet3_drv.c
+--- a/vmxnet3/vmxnet3_drv.c	2018-01-26 08:16:40.302737839 +0300
```

Comment:
This file should be removed from the merge request, since it will be generated from the files in the linux repository.